kura/org.eclipse.kura.core/OSGI-INF/metatype/org.eclipse.kura.data.DataService.xml

```diff
@@ -98,7 +98,25 @@
             default="0"
             min="0"
             description="Timeouts the in-flight messages congestion condition. The service will force a disconnect attempting to reconnect (0 to disable)."/>
+
+        <AD id="max.publish.rate"
+            name="Maximum publish rate"
+            type="Double"
```

Comment:
max.publish.rate is the number of messages that should be send in a second. So, if you want to publish 1 messages every 10 seconds, the value is 0.1. Float is sufficient.
Is there a better way to show it?